[1.26.0RC1] Build failure with PHP 5.5

```
/builddir/build/BUILD/php55-php-pecl-grpc-1.26.0~RC1/NTS/src/php/ext/grpc/php_grpc.c: In function 'custom_logger':
/builddir/build/BUILD/php55-php-pecl-grpc-1.26.0~RC1/NTS/src/php/ext/grpc/php_grpc.c:274:31: error: expected ')' before 'PRId64'
   gpr_asprintf(&prefix, "%s%" PRId64 ".%09" PRId32 " %s:%d]",
                               ^
/builddir/build/BUILD/php55-php-pecl-grpc-1.26.0~RC1/NTS/src/php/ext/grpc/php_grpc.c:276:16: warning: format '%s' expects a matching 'char *' argument [-Wformat=]
                now.tv_nsec, display_file, args->line);
                ^
```

Reported because package.xml state minimal version is 5.5
Perhaps can be raised to 5.6.... 5.5 being EOL for a while (as 5.6, 7.0 and 7.1)

--- a/src/php/ext/grpc/php_grpc.c
+++ b/src/php/ext/grpc/php_grpc.c
@@ -26,6 +26,7 @@
 #include "call_credentials.h"
 #include "server_credentials.h"
 #include "completion_queue.h"
+#include <inttypes.h>
 #include <grpc/support/alloc.h>
 #include <grpc/support/log.h>
 #include <grpc/support/string_util.h>

